### PR TITLE
Remove API key requirement and rebrand to Python API (not Python Clie…

### PR DESCRIPTION
**Pull request by tjann**
…nt API).

### CODE REVIEW

**Comment by tjann** [comment]
Tested in a notebook-- but fixed the date after.

image

File: datacommons/utils.py [hunk 1]
```diff
@@ -57,27 +58,13 @@
...
-
-  For more details about how to get an API key and provide it to the Python
-  Client API, please visit :ref:`getting_started`
-
-  Args:
-    api_key (:obj:`str`): The api key.
-  """
-  os.environ[_ENV_VAR_API_KEY] = api_key
+  """DEPRECATED FUNCTION--API keys no longer required."""
+  logging.warning('Data Commons has removed the API key requirement. This function will be removed by Dec 1st, 2020.')
```
Comment (tjann) [inline datacommons/utils.py:67]: @beets wanted you to review this decision :)

Comment (shifucun) [reply datacommons/utils.py:67]: Lg. Do we have other docs/codes to remove this as well?

Comment (tjann) [reply datacommons/utils.py:67]: Yeah need to update docsite... I think that's it. I went through the api-python repo pretty carefully to remove api key refs.

**Comment by tjann** [comment]
Thanks for review! @beets I can submit a patch if you have any other feedback.

File: datacommons/utils.py (hunk 1 above)
Comment (beets) [reply datacommons/utils.py:67]: Thanks for calling this out!

I don't think we want to remove this completely. We're allowing you to use it without a key, but if you already have one, we should continue allowing you to set it.

**Pull request merged**